platform-api/src/routers/forms.py

```
@@ -0,0 +1,136 @@
+            {"name": "Email", "value": body.email, "inline": True},
+            {"name": "Organization", "value": body.organization or "N/A", "inline": True},
+            {"name": "Message", "value": body.message[:1024]}  # Discord limit is 1024
+        ],
+        "timestamp": datetime.now(timezone.utc).isoformat()
+    }
+
+    try:
+        async with get_client() as client:
+            resp = await client.post(webhook_url, json={"embeds": [embed]})
+            resp.raise_for_status()
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)